agent/websockets/connection.go

```diff
@@ -175,7 +182,7 @@ func (conn *Connection) Close() {
 // SendClientMessage sends the given message to the websocket server.
 //
 // The returned error value is non-nill if the connection has been closed.
-func (conn *Connection) SendClientMessage(msg interface{}) error {
+func (conn *Connection) SendClientMessage(msg interface{}, ver int) error {
```

Comment:
This `ver` argument isn't used (and does not seem necessary)

Reply:
There were merge issues in the last commit. Now fixed in the latest.